examples/feature_demo/gltf_viewer.py

```diff
@@ -60,7 +58,7 @@
 model_list: list = response.json()
 
 # filter out models having "core" in tags
-model_list = [m for m in model_list if "core" in m.get("tags", [])]
+# model_list = [m for m in model_list if "core" in m.get("tags", [])]
```

Comment:
Comment or remove?

Reply:
Ah, I removed the tag filter to display all models in the glTF asset library for easier testing of models related to Iridescence.

Currently, many glTF extensions are not yet implemented, so for now, let's keep only the "core" tag. 
I think once we support more advanced rendering features in the glTF extensions, we will remove this restriction finally.